index.html

```diff
@@ -245,17 +246,17 @@ <h2>Profesional Art Printing Data</h2>
             </section>
             <section class="about">
```

Comment:
Some buttons have failed the button contrast tests. You can use this site to test it. Kindly see the screenshot below for reference
![contrast](https://user-images.githubusercontent.com/35504149/136076629-e9139308-354e-4ea7-9c28-fe62bebf87a6.png)
.

Reply:
![image](https://user-images.githubusercontent.com/71874425/136165524-be984a8e-afec-49c8-867f-4a01d35ea3ac.png)

As we can see above, we solved the failed contrast test yet